src/Impostor.Server/Net/Messages/Rpc/RpcMessage13SendChat.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Hazel;
+using Impostor.Shared.Innersloth.Data;
+
+namespace Impostor.Server.Net.Messages.Rpc
+{
+    internal static class RpcMessage13SendChat
+    {
+
+        public static void Deserialize(MessageReader reader, out int rpcChatLength, out byte[] rpcChatMessage)
+        {
+            rpcChatLength = reader.ReadPackedInt32();
```

Comment:
Pretty sure this could be a `ReadString()`, could it? `ReadString()` also does decodes the message as Unicode, but that shouldn't be a problem.
```csharp
namespace Hazel
{
    public class MessageReader : IRecyclable
    {
    // [snip]
        public string ReadString()
        {
            int len = this.ReadPackedInt32();
            string output = UTF8Encoding.UTF8.GetString(this.Buffer, this.readHead, len);

            this.Position += len;
            return output;
        }
    // [snip]
    }
}
```